mailpoet/assets/js/src/automation/editor/components/inserter/index.tsx

```diff
@@ -20,134 +20,139 @@ const filterItems = (value: string, item: Item[]): Item[] =>
 
 type Props = {
   onInsert?: (item: Item) => void;
+  showInserterHelpPanel?: boolean;
```

Comment:
I only added this and used it on [line 27](https://github.com/mailpoet/mailpoet/pull/4634/files#diff-78843e796e9514ac381ab71d1aa0b9353a59faef28119f1626e414c5a0a0f680R27) and [line 152](https://github.com/mailpoet/mailpoet/pull/4634/files#diff-78843e796e9514ac381ab71d1aa0b9353a59faef28119f1626e414c5a0a0f680R152). The rest of the diff is Prettier changing indentation for some reason (the component is wrapped in `forwardRef` and adding a property caused this).

Reply:
For the property, I reused [the naming that's used in the block library](https://github.com/mailpoet/mailpoet/blob/c214eb4c8540f8baae4ae64200b64ebe5c10e289/mailpoet/assets/js/src/form_editor/components/inserter.tsx#L23).